tools/trace2syz/trace2syz/vmaCallHandlers.go

```diff
@@ -0,0 +1,357 @@
+package trace2syz
+
+import (
+	//"fmt"
+	"github.com/google/syzkaller/pkg/log"
+	"github.com/google/syzkaller/prog"
+)
+
+const (
+	pageSize   = 4096
+	MapFixed   = "MAP_FIXED"
+	RemapFixed = "MREMAP_FIXED"
+)
+
+func ParseMemoryCall(ctx *Context) *prog.Call {
+	syzCall := ctx.CurrentSyzCall
+	straceCall := ctx.CurrentStraceCall
+	if straceCall.CallName == "mmap" {
```

Comment:
this looks like a good candidate for `switch straceCall.CallName { ...`

Reply:
sure